jets/e/secp.c

```diff
@@ -0,0 +1,356 @@
+/* j/5/secp.c
+**
+*/
+#include "all.h"
+#include "../include/secp256k1.h"
+#include "../include/secp256k1_recovery.h"
+
+// end-to-end test:
+//   =hash 123
+//   =priv 99
+//   =nonce (make-k:secp256k1:secp:crypto `@uvI`hash priv)
+//   =sig (ecdsa-raw-sign:secp256k1:secp:crypto `@uvI`hash priv)
+//   =reco (ecdsa-raw-recover:secp256k1:secp:crypto `@uvI`hash v:sig  r:sig s:sig)
```

Comment:
This can probably be removed.

Reply:
O_O

If you say so.

Reply:
These are just to assist in testing the jet, right? Or is there something I'm not understanding about this?